ush/parsing_namelists_FV3.sh

```diff
@@ -474,17 +474,16 @@ local MOM6_RESTART_SETTING=${MOM6_RESTART_SETTING:-'n'}
 local MOM6_RESTART_DIR=MOM6_RESTART/
 
 # Land IAU defaults
-
-local DO_LAND_IAU=.false.
-local LAND_IAU_FHRS=3,6,9
-local LAND_IAU_DELHRS=6
-local LAND_IAU_INC_FILES="'sfc_inc',''"
-local LSOIL_INCR=3
-local LAND_IAU_FILTER_INC=.false.
-local LAND_IAU_UPD_STC=.true.
-local LAND_IAU_UPD_SLC=.true.
-local LAND_IAU_DP_STCSMC_ADJ=.true.
-local LAND_IAU_MIN_T_INC=0.0001
+local DO_LAND_IAU=${DO_LAND_IAU:-".false."}
+local LAND_IAU_FHRS=${LAND_IAU_FHRS}
+local LAND_IAU_DELHRS=${LAND_IAU_DELHRS}
+local LAND_IAU_INC_FILES=${LAND_IAU_INC_FILES:-"'sfc_inc',''"}
+local LSOIL_INCR=${LSOIL_INCR}
+local LAND_IAU_FILTER_INC=${LAND_IAU_FILTER_INCREMENTS:-".false."}
+local LAND_IAU_UPD_STC=${LAND_IAU_UPD_STC}
+local LAND_IAU_UPD_SLC=${LAND_IAU_UPD_SLC}
+local LAND_IAU_DP_STCSMC_ADJ=${LAND_IAU_DO_STCSMC_ADJUSTMENT}
+local LAND_IAU_MIN_T_INC=${LAND_IAU_MIN_T_INCREMENT}
```

Comment:
```suggestion
local DO_LAND_IAU=${DO_LAND_IAU:-".false."}
local LAND_IAU_FHRS=${LAND_IAU_FHRS}
local LAND_IAU_DELHRS=6
local LAND_IAU_INC_FILES=${LAND_IAU_INC_FILES:-"'sfc_inc',''"}
local LSOIL_INCR=${LSOIL_INCR}
local LAND_IAU_FILTER_INC=${LAND_IAU_FILTER_INCREMENTS:-".false."}
local LAND_IAU_UPD_STC=".true."
local LAND_IAU_UPD_SLC=".true."
local LAND_IAU_DP_STCSMC_ADJ=".true."
local LAND_IAU_MIN_T_INC=0.001
```
Since these are fixed, they can be set as defaults here and removed from `config.base`

Reply:
These are not necessarily fixed. We are still testing with different settings (e.g., only updating temp and not moisture) so better to keep them at least for now.

Reply:
understood.